test.js

```diff
@@ -5,7 +5,7 @@ var assert = require('assert');
 Object.assign = undefined;
 var objectAssign = require('./');
 
-it('should have the correct length', function () {
+it('should expect at least 2 arguments', function () {
```

Comment:
Hmm... from [mdn](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Function/length), it should mean the number of arguments expected by the function.